Makefile

```diff
@@ -153,7 +157,7 @@ utils_wrapped: $(BINS)
 $(SHARED_LIB): $(OBJS_LIB) $(MAP_LIB)
 	$(LD_lib) $(LDFLAGS_lib) $(OBJS_LIB) $(LDLIBS_lib) -o $(SHARED_LIB)
 
-$(DEVEL_LIB): $(SHARED_LIB)
+$(DEVEL_LIB): $(SHARED_LIB) $(PKGCONFIG)
```

Comment:
Does `$(DEVEL_LIB)` really depend on `$(PKGCONFIG)`?

Reply:
Well probably not directly but in my opinion passwdqc.pc should be generated each time dev library is created. Is there a better way to do that?

Reply:
Unlike `$(DEVEL_LIB)`, `$(PKGCONFIG)` is not needed to build anything else.
Maybe just add `$(PKGCONFIG)` to `all_wrapped`?

Reply:
> `$(PKGCONFIG)` is not needed to build anything else.

That make point, will add it to `all_wrapped`